src/Services/AppointmentService.php

```diff
@@ -84,17 +92,41 @@ public function getUuidFields(): array
         return ['puuid', 'pce_aid_uuid', 'pc_uuid', 'facility_uuid', 'billing_location_uuid' ];
     }
 
+    /**
+     * Validates the data for updating an appointment.
+     *
+     * @param array $appointment An array containing the appointment data to be validated.
+     * @return ValidationResult Returns the result of the validation operation.
+     */
+    public function update_validate(array $appointment)
```

Comment:
These methods could go in another class.
Validation.php
class Validation {
    public function update_validate(array $appointment) {...}

}

And later you can add this class on the constructor method of this class [AppointmentService]